chesweet/test/test_chesweet.py

```diff
@@ -161,6 +161,20 @@ def test_round_down_up():
     for idx, t in enumerate(torsionals):
         assert _round_down_up(t, 10) == ref[idx]
 
+def test_compute_cs_all_no_path():# load all without path and check if this run
```

Comment:
Why we need to test, for _all_, _few_ and _one_ as separated cases?

Reply:
Maybe the test for *one* is unnecessary because use the same part of the function that is used to load a *few* disaccharides, so we can eliminate that test. But to load *all* the disaccharides another part of the function is used, because of that is a different test.
In the other hand, perhaps will be useful make 2 more test with the option `full=True` to load the full lut, what do you think?.

Reply:
OK I see, but now it is not clear to me why we need these specific tests if we already have `test_compute_cs_full` and `test_compute_cs_red`...

Reply:
*For the record* :microphone: When the module is load you have 3 arguments that you can set, each argument has 2 option, and when set different combination of these arguments works different parts of the code. For that, there are the different tests for the different possible combination at the moment to load the module.